addons/sys_rpc/fnc_handleData.sqf

```diff
@@ -6,7 +6,7 @@
  * 0: Message <STRING>
  *
  * Return Value:
- * Always returns true <BOOL>
+ * Handled - Returns true if a handler was called. <BOOL>
```

Comment:
`* Handled (returns true if a handler was called) <BOOL>`

Full stops in lists look ugly.